content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm

```diff
@@ -54,7 +104,7 @@ includeLatestJQuery('sakai_filepicker_attach.vm');
 
 <div class="portletBody specialLink" id="portletBody">
 	#if ($alertMessage)
-		<div class="alertMessage">$tlang.getString("gen.alert") $formattedText.escapeHtml($alertMessage)</div><div style="display:block;clear:both" ></div>
+		<div id="alertMessage" class="sak-banner-error" style="display:none;">$tlang.getString("gen.alert")</div><div style="display:block;clear:both" ></div>
```

Comment:
```suggestion
		<div id="alertMessage" class="sak-banner-error d-none">$tlang.getString("gen.alert")</div><div class="clearfix"></div>
```
We don't have to use inline styles here, we can use bootstrap-5 class `d-none`. 
I have also replaced the second div with Bootstrap's `clearfix` utility, which clears floated content. However, with modern flexbox and grid layouts in Bootstrap 5, you might not even need clearfixes anymore, depending on your layout.

Reply:
suggestion completed